src/schema/v2/me/secondFactors/secondFactors.ts

```diff
@@ -12,6 +12,7 @@ import {
 } from "graphql"
 import { date } from "../../fields/date"
 import { ResolverContext } from "types/graphql"
+import { ErrorsType } from "lib/gravityErrorHandler"
```

Comment:
btw @mzikherman I wanted to check wether I broke something in second factors with this change and I've noticed that even on staging error handling doesn't work as it should. I'm getting such error:

```json
{
  "errors": [
    {
      "message": "Cannot return null for non-nullable field Error.message.",
      "locations": [
        {
          "line": 10,
          "column": 11
        }
      ],
      "path": [
        "deliverSecondFactor",
        "secondFactorOrErrors",
        "errors",
        0,
        "message"
      ]
    }
  ]
}
```

for such request:

```graphql
mutation {
  deliverSecondFactor(input: { secondFactorID: "does not exist" }) {
    secondFactorOrErrors {
      ... on SmsSecondFactor {
        phoneNumber
      }
      
      ... on Errors {
        errors {
          message
        }
      }
    }
  }
}
```

Reply:
Likely something in the error parsing for a 404 - will check it out! That shouldn't happen in prod (as opposed to more likely error states like incorrect password/OTP which should be working - thanks for flagging!